src/main/java/com/google/identitytoolkit/GitkitClient.java

```diff
@@ -176,6 +176,79 @@ public GitkitUser validateTokenInRequest(HttpServletRequest request)
     }
     return null;
   }
+  
+  /**
+   * Verifies the user entered password at Gitkit server.
+   *
+   * @param email The email of the user
+   * @param password The password inputed by the user
+   * @param pendingIdToken The GITKit token for the non-trusted IDP, which is to be confirmed by the user
+   * @param captchaChallenge The captcha challenge
+   * @param captchaResponse Response to the captcha
+   * @return the verify password response.
+   * @throws GitkitClientException for invalid request
+   * @throws GitkitServerException for server error
+   */
+  public VerifyPasswordResponse verifyPassword(String email, String password, String pendingIdToken,
```

Comment:
It is worth to add another verifyPassword(email, password), which will be the most frequent use case.